<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeButtons</class>
 <widget class="QWidget" name="TimeButtons">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="button_StepDecrease">
       <property name="toolTip">
        <string>Decrease Step</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="simQtResources.qrc">
         <normaloff>:/simQt/images/Navigation Blue Left.png</normaloff>:/simQt/images/Navigation Blue Left.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="button_StepBack">
       <property name="toolTip">
        <string>Frame Reverse</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="simQtResources.qrc">
         <normaloff>:/simQt/images/Navigation Blue First.png</normaloff>:/simQt/images/Navigation Blue First.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="button_PlayBackwards">
       <property name="toolTip">
        <string>Replay</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="simQtResources.qrc">
         <normaloff>:/simQt/images/Navigation Blue Previous.png</normaloff>:/simQt/images/Navigation Blue Previous.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="button_StartStop">
       <property name="toolTip">
        <string>Stop</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="simQtResources.qrc">
         <normaloff>:/simQt/images/Navigation Blue Next.png</normaloff>:/simQt/images/Navigation Blue Next.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="button_Stop">
       <property name="icon">
        <iconset resource="simQtResources.qrc">
         <normaloff>:/simQt/images/Navigation Blue Stop.png</normaloff>:/simQt/images/Navigation Blue Stop.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="button_Play">
       <property name="icon">
        <iconset resource="simQtResources.qrc">
         <normaloff>:/simQt/images/Navigation Blue Next.png</normaloff>:/simQt/images/Navigation Blue Next.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="button_Step">
       <property name="toolTip">
        <string>Frame Forward</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="simQtResources.qrc">
         <normaloff>:/simQt/images/Navigation Blue Last.png</normaloff>:/simQt/images/Navigation Blue Last.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="button_StepIncrease">
       <property name="toolTip">
        <string>Increase Step</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="simQtResources.qrc">
         <normaloff>:/simQt/images/Navigation Blue Right.png</normaloff>:/simQt/images/Navigation Blue Right.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="button_Realtime">
       <property name="toolTip">
        <string>Real Time</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="simQtResources.qrc">
         <normaloff>:/simQt/images/Symbol Clock.png</normaloff>:/simQt/images/Symbol Clock.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="simQtResources.qrc"/>
 </resources>
 <connections/>
</ui>
